src/scripts/renderForm.js

```diff
@@ -0,0 +1,17 @@
+const paginationComponent = document.querySelector('.pagination-component');
+
+const renderForm = () => {
+    const input = /* HTML */ `
+        <form action="" method="GET" class="pagination-component__form">
+            <label for="pagination-component__input">Max page:</label>
+            <input
+                id="pagination-component__input"
```

Comment:
pytanie czy ID powinno mieć to co BEMowa klasa czy nie coś bardziej definiującego element jako jedyny na stronie